pkg/gql/query/types.go

```diff
@@ -36,7 +36,10 @@ var Header = graphql.NewObject(
 		Name: "Header",
 		Fields: graphql.Fields{
 			"height": &graphql.Field{
-				Type: graphql.Int,
+				Type: graphql.Float,
```

Comment:
Is height now a float?

Reply:
I forgot to mention that `graphql.Float` is used to map `uint64`, otherwise if the value overflows the `Int.MaxValue` it returns `0`
See also #1315